src/coreclr/jit/emitxarch.cpp

```diff
@@ -257,6 +263,67 @@ bool emitter::IsEvexEncodableInstruction(instruction ins) const
     return HasEvexEncoding(ins);
 }
 
+//------------------------------------------------------------------------
+// IsRex2EncodableInstruction: Answer the question- Can this instruction be Rex2 encoded.
+//
+// Arguments:
+//    ins - The instruction to check.
+//
+// Returns:
+//    `true` if ins can be Rex2 encoded.
+//
+bool emitter::IsRex2EncodableInstruction(instruction ins) const
+{
+    // TODO-Xarch-apx: we have special stress mode for REX2 on non-compatible machine, that will
+    //                 force UseRex2Encoding return true regardless of the CPUID results.
```

Comment:
Do we need this comment? We don't have a corresponding one for `IsEvexEncodableInstruction`

(same general question/concept applies throughout; I think we can generally mirror the comments/semantics that `EVEX` and `stress EVEX` has)

Reply:
> (same general question/concept applies throughout; I think we can generally mirror the comments/semantics that `EVEX` and `stress EVEX` has)

Yes, the overall concept of the REX2 stress is mirrored from EVEX stress mode, thanks for the inputs above, I will try to do the improvement to make the REX2 stress mode more aligned with the current stress mode design like EVEX stress mode does.

Reply:
I think this comment is still unnecessary, as we don't have such special comments about other stress modes.

Reply:
comment removed.